src/planner/plan_builder.rs

```diff
@@ -1,13 +1,16 @@
 use crate::errors::{MySQLError, MySQLResult};
-use crate::planner::point_get_plan::QueryPlanBuilder;
+use crate::expression::Expr;
```

Comment:
There is `crate::planner::plan_expression`. Is it too hurry to convert AST to executable expression?

Reply:
> There is `crate::planner::plan_expression`. Is it too hurry to convert AST to executable expression?

Is there? I haven't seen that before.

By the way, this `Expr` is just a descriptor of expression, which is decoupled from execution. So I think it's fine to do this.